frontend/javascripts/components/secured_route.tsx

```diff
@@ -53,21 +64,35 @@ class SecuredRoute extends React.PureComponent<SecuredRouteProps, State> {
       <Route
         {...rest}
         render={(props) => {
-          if (isCompletelyAuthenticated) {
-            if (Component != null) {
-              return <Component />;
-            } else if (render != null) {
-              return render(props);
-            } else {
-              throw Error("Specified neither component nor render function.");
-            }
+          if (!isCompletelyAuthenticated)
+            return <LoginView redirect={this.props.location.pathname} />;
+
+          if (
+            this.props.requiredPricingPlan &&
+            this.props.activeOrganization &&
+            !isPricingPlanGreaterEqualThan(
+              this.props.activeOrganization.pricingPlan,
+              this.props.requiredPricingPlan,
+            )
```

Comment:
In your case, if I am logged out (activeOrganization = undefined ) I will be redirected to <PageUnavailableForYourPlanView>. Does it not make more sense to be redirected to a login instead?

Reply:
shouldn't `isCompletelyAuthenticated` be false when being logged out so that the log in view is shown by line 68? my reasoning for the suggestion was that `<Component />` in line 78 should not be rendered if `activeOrganization` is false-y and `requiredPricingPlan` truthy.

Reply:
Done